### PR TITLE
solution

### PR DESCRIPTION
**Pull request by tykhan**
[DEMO LINK](https://tykhan.github.io/react_movies-list-filter/)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -22,12 +26,16 @@ export class App extends Component {
                   id="search-query"
                   className="input"
                   placeholder="Type search word"
+                  // value={this.state.query}
```
Comment (Andrii256) [inline src/App.js:29]: ?

File: src/App.js [hunk 2]
```diff
@@ -22,12 +26,16 @@ export class App extends Component {
...
+                  onChange={(event) => {
+                    this.setState({ query: event.target.value });
+                  }}
                 />
               </div>
             </div>
           </div>
 
-          <MoviesList movies={moviesFromServer} />
+          <MoviesList movies={moviesFromServer} query={query} />
```
Comment (Andrii256) [inline src/App.js:38]: хм, а це ідея

**Review by Andrii256 — APPROVED** [review]

File: src/components/MoviesList/MoviesList.js [hunk 3]
```diff
@@ -4,15 +4,23 @@ import PropTypes from 'prop-types';
...
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} {...movie} />
-    ))}
-  </div>
-);
+export const MoviesList = ({ movies, query }) => {
+  const visibleMovies = movies.filter(
+    movie => movie.title.toLowerCase().includes(query.toLowerCase())
+      || movie.description.toLowerCase().includes(query.toLowerCase()),
```
Comment (Denchepornyuk) [inline src/components/MoviesList/MoviesList.js:10]: (title + description).toLowerCase().includes(...)

**Review by Denchepornyuk — APPROVED** [review]

File: src/components/MoviesList/MoviesList.js (hunk 3 above)
Comment (tykhan) [reply src/components/MoviesList/MoviesList.js:10]: wow, didn't know about this possibility👍

**Pull request closed without merging**